index.src.html

```diff
@@ -633,6 +631,10 @@ <h3 id="set-referrer-policy">
 
```

Comment:
Nit: I'd suggest updating this text as well. It might be worthwhile to add an "authoring considerations" section somewhere as well noting that y'all intentionally decided upon this behavior when policy is set globally for a document from multiple sources. Maybe with examples. :)